tuichain/api/models.py

```diff
@@ -81,17 +80,18 @@ def to_dict(self):
             "school": self.school,
             "course": self.course,
             "destination": self.destination,
-            "amount": self.amount,
-            "current_amount": self.current_amount,
+            "requested_value_atto_dai": self.requested_value_atto_dai,
             "description": self.description,
-            "status": self.status,
+            "state": str(LoanState(self.state)),
```

Comment:
This dictionary is used in several places as part of responses, but there the state should also reflect the phase of the loan in the blockchain.

Reply:
I think it would make more sense for us to pass de phase of the loan not in that dictionary but in the response itself with a different name

Reply:
Sure, we can do that in a future PR.